index.html

```diff
@@ -151,6 +244,115 @@ <h2 class="works-2-div2-1-a-p">Multi-Post<br>Stories</h2>
                     </button>
                 </div>
             </div>
+        </div>
+                <!--works2-->
+        <div class="works-2">
+            <div class="works-2-div1">
+                <img class="works-2-div1-img" src="images/Snapshoot Portfolio2.svg" alt="">
+                <img class="works-2-div1-img-desktop" src="images/Snapshoot Portfoliowork2.svg" alt="">
+            </div>
+            <div class="works-2-div2">
+                <div class="works-2-div2-1">
+                    <div class="works-2-div2-1-a">
+                        <h2 class="works-2-div2-1-a-p">Multi-Post<br>Stories</h2>
+                    </div>
+                    <div class="works-2-div2-1-b">
+                        <ul class="works-2-div2-1-b-ul">
+                            <li class="works-2-div2-1-b-ul-li-1">
+                                <span class="works-2-div2-1-b-ul-li-1-sp">CANOPY</span>
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-2">
+                                <img class="works-2-div2-1-b-ul-li-2-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-3">
+                                <span class="works-2-div2-1-b-ul-li-3-sp">Back End Dev</span>
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-4">
+                                <img class="works-2-div2-1-b-ul-li-4-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-2-div2-1-b-ul-li-5">
+                                <span class="works-2-div2-1-b-ul-li-5-sp">2015</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="works-2-div2-2">
+                    <p class="works-2-div2-2-p">A daily selection of privately 
+                        personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="works-2-div2-3">
+                    <li class="works-2-div2-3-btn1" type="button">
+                        <span class="works-2-div2-3-btn1-sp1">html</span>
+                    </li>
+                    <li class="works-2-div2-3-btn2" type="button">
+                        <span class="works-2-div2-3-btn2-sp2">css</span>
+                    </li>
+                    <li class="works-2-div2-3-btn3" type="button">
+                        <span class="works-2-div2-3-btn3-sp3">javaScript</span>
+                    </li>
+                </ul>
+                <div class="works-2-div2-4">
+                    <button class="works-2-div2-4-btn" type="button">
+                        <span class="works-2-div2-4-btn-sp">See Project</span>
+                    </button>
+                </div>
```

Comment:
- [ ] [**OPTIONAL**] Kindly make sure you add to the button the `cursor:pointer` property, so the user will know they are clickable :clap: